packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -106,10 +109,11 @@ class LinkViewController extends PlatformViewController {
       // This is the first controller being created, attach the global click
       // listener.
 
-      _clickSubscription =
-          const html.EventStreamProvider<html.MouseEvent>('click')
-              .forTarget(html.window)
-              .listen(_onGlobalClick);
+      // We listen to `keydown` events in the capture phase to ensure that we
+      // receive the event even if the engine calls `stopPropagation`.
+      html.window
+          .addEventListener('keydown', _jsGlobalKeydownListener, _useCapture);
```

Comment:
Can this be attached to the `<a>` element that is backing the widget, instead of the global window?

Reply:
See [this comment](https://github.com/flutter/packages/pull/6505#discussion_r1562659976) for context.

Long story short, when the user is tabbing through the Flutter UI elements, only Flutter is tracking where focus is; the browser has no idea which element is focused (in many cases, it can't because the focused button may be drawn on a `<canvas>`). From a browser's perspective, most of the time, the focus is on `<flutter-view>` (with some exception like text fields). So key events are sent to `<flutter-view>`, not to the actual button or `<a>` element.

Reply:
I added this explanation to the code comments.